pkg/metrics/metrics.jsx

```diff
@@ -390,20 +404,33 @@ class CurrentMetrics extends React.Component {
                 </Tooltip>);
         }
 
+        const top_cores = this.state.cpuCoresUsed.map((v, i) => [i, v]).sort((a, b) => b[1] - a[1])
+                .slice(0, 16);
+        const cores = top_cores.map(c => <Flex key={c[0]} justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+            <FlexItem>{ cockpit.format(_("Core $0"), c[0]) }</FlexItem>
+            <FlexItem>{c[1]}%</FlexItem></Flex>
+        );
```

Comment:
This could be conditionalized on `this.state.cpuCoresUsed.length > 1`, on single-core it's just wasted electrons.